scripts/Engineering/gui/engineering_diffraction/engineering_diffraction.py

```diff
@@ -45,23 +46,27 @@ def __init__(self, parent=None):
         self.fitting_presenter = None
         self.settings_presenter = None
         self.calibration_observer = CalibrationObserver(self)
+        self.savedir_observer = SavedirObserver(self)
         self.set_on_help_clicked(self.open_help_window)
 
         self.set_on_settings_clicked(self.open_settings)
         self.btn_settings.setIcon(get_icon("mdi.settings", "black", 1.2))
 
-        # Setup status bar
-        self.status_label = QtWidgets.QLabel()
-        self.setup_statusbar()
-
         # Setup Elements
         self.setup_settings()
         self.setup_calibration()
         self.setup_focus()
         self.setup_fitting()
 
+        # Setup status bar
+        self.status_label = QtWidgets.QLabel()
+        self.savedir_label = QtWidgets.QLabel()
+        self.savedir_label.setMaximumWidth(300)
```

Comment:
Definitely a nitpick, but would it be preferable to declare a const elsewhere for the max width pixels and avoid this magic number?

Reply:
Done!